paper-toast.html

```diff
@@ -94,28 +102,32 @@
        */
       text: {
         type: String,
-        value: ""
+        value: ''
       },
 
       /**
-       * True if the toast is currently visible.
+       * Set to false to enable closing of the toast by clicking outside it.
        */
-      visible: {
+      noCancelOnOutsideClick: {
```

Comment:
@cdata, I was undecided between exposing this property, or override `_onCaptureClick` with an empty function. Exposing the property though gives more freedom to the user